### PR TITLE
initialize book store project

### PR DESCRIPTION
**Pull request by aron-net**
- display the list of books with title and author
- form component with inputs for title and author
-  categories with select and check status button.

### CODE REVIEW

File: src/Components/pages/Books.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import BookList from '../Book/BookList';
+import NewBook from '../NewBook/NewBook';
+/* eslint-disable react/prop-types */
+
```
Comment (codepantha) [inline src/Components/pages/Books.js]: - [ ] Please don't disable linters; they are there to help you write better code.

File: src/Components/pages/Books.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <BookList key={item.id} title={item.title} author={item.author} />
+    ))}
+    ;
+    <NewBook />
+  </ul>
+);
+
+// Book.propTypes = {
+//   items: PropTypes.string.isRequired,
+// };
```
Comment (codepantha) [inline src/Components/pages/Books.js]: - [ ] Please implement the prop-types as they are a kind of guard that checks the props passed to your component at runtime 🙏

File: src/Components/pages/Categories.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import Category from '../Book/Category';
+
+const Categories = () => (
+  <div>
+    <Category />
+    <button type="button">Check status</button>
+  </div>
```
Comment (codepantha) [inline src/Components/pages/Categories.js]: - [ ] According to requirements, the categories should display only a button with text "Check status", so please remove the `<Category />` you have on line 5 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi @aron-helu 👋, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please modify your README file to describe your project better and be more professional 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️ 


@aron-helu  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] Like the last code reviewer said kindly update your README file. 
You can use this [template](https://github.com/microverseinc/readme-template) 
and make sure there are no broken links(screenshot, license, etc.) and empty sections. 
You should change the readme template's sentence with your sentence.
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by Lordkaito — APPROVED** [review]
Hi @aron-helu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**